tests/run_code_style.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -xeu
+
+if [ $1 = "lint" ]; then
+    flake8 ignite tests examples --config setup.cfg
+    isort . --check --settings setup.cfg
+    black . --check --config pyproject.toml
+elif [ $1 = "fmt" ]; then
+    flake8 ignite tests examples --config setup.cfg
```

Comment:
@ydcjeff this can lead to incorrect behaviour when flake8 fails and script will exit here due to `set -xeu`.
Why you replaced `autopep8 --recursive --in-place --aggressive --aggressive .` by flake8 ?
Let's restore previous behaviour with autopep8. Please, send a PR with the fix. Thanks

Reply:
`autopep8` is a formatter. We already have `black`. So we probably don't need two formatters.

Reply:
That's true about `autopep8`, I agree with you, let's keep only black then and remove `flake8` as if it fails script wont execute the following below commands. Thanks for pointing out, @ydcjeff  !